lstchain/io/io.py

```diff
@@ -443,6 +445,239 @@ def read_array_info(filename):
     return array_info
 
 
+def read_optic(filename, telescope_name):
```

Comment:
Function name is not so nice, and a bit too close to the other one. Any better option?
e.g. read_single_optics    (and keep read_optics as it is)

Reply:
And thanks a lot for this! Will try it.

Reply:
Good point, I will rename this one and similar with `_single_`